schedule/sle-perf/hanaperf_installation.yaml

```diff
@@ -0,0 +1,59 @@
+---
+name: hanaperf_installation
+description: >
+  OS installation for HANA performance
+vars:
+  MULTIPATH: '1'
+  MULTIPATH_CONFIRM: 'yes'
+  SEPARATE_HOME: '1'
+  NUMDISKS: '7'
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - '{{scc_registration}}'
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_smalldisk_storageng
+  - '{{separate_home}}'
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - boot/reconnect_mgmt_console
+  - support_server/login
+  - kernel_performance/install_qatestset
+  - boot/reconnect_mgmt_console
+  - support_server/login
+  - kernel_performance/full_run
+conditional_schedule:
+  scc_registration:
+    SCC_REGISTER:
+      none:
+        - '{{multipath}}'
+        - installation/scc_registration
+      installation:
+        - installation/scc_registration
+        - '{{multipath}}'
```

Comment:
Are you sure that in case when test does not register the system, the multipath popup appeared before Registration screen, but in case of registering it does not appear before the Registration screen, but appears after it?

Could you please run VR to check this. I guess the behavior may be the same.

Reply:
This may be specific by our test cases , we are run Online testing for 15SP3 (SCC_REGISTER=installation), Full testing for 15SP2(SCC_REGISTER=none), use this condition to split the different workflow.
15-SP2 FULL http://10.67.19.72/tests/3206
15-SP3 Online http://10.67.19.72/tests/3207